odpf/optimus/core/v1beta1/runtime.proto

```diff
@@ -559,13 +568,26 @@ message RegisterSecretRequest {
     string project_name = 1;
     string secret_name = 2;
     string value = 3; // base64 encoded secret value
+    string namespace_name = 4;
 }
 
 message RegisterSecretResponse {
     bool success = 1;
     string message = 2;
 }
 
+message UpdateSecretRequest {
+    string project_name = 1;
+    string secret_name = 2;
+    string value = 3; // base64 encoded secret value
+    string namespace_name = 4;
+}
+
+message UpdateSecretResponse {
+    bool success = 1;
+    string message = 2;
```

Comment:
How are we using these? I don't think we need a success flag here?

Reply:
Sure, will remove `success` flag from the proto in next PR.